src/sdk/pynni/nni/metis_tuner/metis_tuner.py

```diff
@@ -245,7 +250,7 @@ def receive_trial_result(self, parameter_id, parameters, value):
 
             # calculate y aggregation
             median = get_median(temp_y)
-            self.samples_y_aggregation[idx] = median
+            self.samples_y_aggregation[idx] = [median]
```

Comment:
so this solves the ValueError of "setting an array element with a sequence"?

Reply:
yes